### ISSUE TITLE
Selection box color

### ISSUE THREAD

**Issue by tevert** [dg9ngf/MultiSelectTreeView#30]
There are two DependencyProperties on MultiSelectTreeView that allow you to change the color of the selection box and border. 

However, their ownerType is MultiSelectTreeViewItem, and whenever I attempt to style the property I get an error to the tune of "MultiSelectTreeView does not have this property". Can you verify that this is correct? What is the correct way to set those properties? 

https://github.com/dg9ngf/MultiSelectTreeView/blob/master/MultiSelectTreeView/Controls/MultiSelectTreeView.cs#L27

https://github.com/dg9ngf/MultiSelectTreeView/blob/master/MultiSelectTreeView/Controls/MultiSelectTreeView.cs#L33

Thanks!

**Comment by dg9ngf** [comment]
Well, would it work to change the owner from the tree view item to the tree view itself?

**Comment by tevert** [comment]
Yes, I'm pretty sure. It looks like a copy-paste error; somebody moved those properties from the treeview item to the treeview without updating them.
